File: backend/nightbiteApi/src/main/java/ni/edu/uam/nightbiteapi/dto/MessageResponse.java
package ni.edu.uam.nightbiteapi.dto;

/**
 * DTO utilizado para enviar mensajes simples desde la API.
 *
 * Se usa principalmente para respuestas de error o confirmaciones.
 */
public class MessageResponse {

    private String message;

    public MessageResponse() {
    }

    public MessageResponse(String message) {
        this.message = message;
    }

    public String getMessage() {
        return message;
    }
}

File: mobile/nightbiteApp/app/src/main/java/ni/edu/uam/nightbiteapp/data/remote/dto/MessageResponse.kt
package ni.edu.uam.nightbiteapp.data.remote.dto

/**
 * DTO utilizado para leer mensajes simples enviados por la API.
 *
 * Se usa principalmente para errores de validación y respuestas
 * informativas del backend.
 */
data class MessageResponse(
    val message: String
)